src/manage-apps/manage-repos.md

```diff
@@ -0,0 +1,78 @@
+# Managing Repositories
+
+Add, remove, and edit repositories using Repoman. Repoman allows users to add Personal Package Archives (PPAs) and Flatpak repositories. Users may want to add a PPA or additional Flatpak repository if their software is not included in the Pop!\_Shop, or they may want to remove PPA’s after uninstalling a package.
+
+>**Caution**: Adding PPA’s and additional Flatpak repositories allows installation of software that has not been validated by System76 or other trusted Linux repositories. Software available via PPA’s and third party Flatpak repositories may not be vetted against packages that introduce security vulnerabilities. Users should take caution and only add external repositories that they trust.
+---
+
+## Managing Debian Repositories (PPAs) with Repoman
+
+PPAs, or Debian archives, are created by developers to distribute software not included in default Debian repositories.
```

Comment:
> PPAs, or Debian archives,

See above; it's perfectly fine to mention PPAs since that is a term Ubuntu users may be familiar with (or that Ubuntu guides may refer to), but they are not the same thing. If anything, this would be `Debian repositories (including PPAs)`.